src/oaklib/implementations/sparql/abstract_sparql_implementation.py

```diff
@@ -388,7 +388,7 @@ def all_rdflib_triples(self) -> Iterator[RDF_TRIPLE]:
             raise NotImplementedError("Not implemented for SPARQL endpoints")
         yield from self.graph.triples((None, None, None))
 
-    def hierararchical_parents(self, curie: CURIE, isa_only: bool = False) -> List[CURIE]:
+    def hierarchical_parents(self, curie: CURIE, isa_only: bool = False) -> List[CURIE]:
```

Comment:
As an aside: some more care is needed this kind of thing does not happen.. :P Sorry its unrelated. But I keep hearing from people like Anita that are relying heavily on these implementations that things change a lot from one version to another; I guess this is still ok right now but are using OAK more and more in production pipelines now.

Reply:
understood. These should only occur between middle digit versions; this one is such a clear error it seemed reasonable to include this but there is an argument for upping the middle digit next release.